### ISSUE TITLE
Footnotes missing

### ISSUE THREAD

**Issue by sukima** [madmalik/mononoki#105]
I found that the following do not display properly on Windows 11

* ¹ Dec 185, Hex 00b9, Oct 271
* ² Dec 178, Hex 00b2, Oct 262
* ³ Dec 179, Hex 00b3, Oct 263
* ⁴ Dec 8308, Hex 2074, Oct 20164
* ⁵ Dec 8309, Hex 2075, Oct 20165

Unfortunately, I tend to use these for footnotes in UTF-8 files. Any ideas?

![MononokiSnap](https://user-images.githubusercontent.com/70075/222596779-09c22b09-f5c8-4638-92c5-960a0055e39e.png)

**Comment by felis-blue** [comment]
Are you using the most recent version (1.6) of mononoki? And is this happening with the OTF, TTF or both?

**Comment by sukima** [comment]
I downloaded from this link: https://github.com/madmalik/mononoki/releases/download/1.6/mononoki.zip

First I installed the four TTF fonts and noticed this issue. I then uninstalled mononoki and reinstalled the four OTF fonts and same result.

I used Notepad this time and the `Consolas` font for comparison.

![MononokiNotepad](https://user-images.githubusercontent.com/70075/222606536-3d5ee195-2b36-4131-8951-9291a187fc15.png)
![MononokiNotepadConsolas](https://user-images.githubusercontent.com/70075/222606537-0fe27e47-8180-4fb0-aba7-e899f327e0d3.png)

**Comment by madmalik** [comment]
I've just booted up Windows and tested it - those characters show up for me:
![image](https://user-images.githubusercontent.com/1328396/222710960-61e97a34-bcbf-4519-9e36-09e9616c0b62.png)

This is under Windows 10, not 11, but it would be very weird if that would make a difference here. Tbh, I'm stumped. If you go to the [website](https://madmalik.github.io/mononoki/) and scroll down to the overview, does your browser show those glyphs?

**Comment by madmalik** [comment]
@sukima Did you have a chance to check this?

**Comment by felis-blue** [comment]
Closing this since we were unable to reproduce the issue.

@sukima Please comment here with the requested info if you get the chance and we can reopen the issue and look into it further!

**Comment by sukima** [comment]
Yeah, the browser does show those Glyphs. Crap. I don't understand why my terminal and also Notepad.exe do not.

**Comment by sukima** [comment]
Looking further it seems when I install mononoki on my system that it either misses or removes those characters. My charmap has none of those symbols.

![Charmap](https://user-images.githubusercontent.com/70075/229000793-e37e79c5-d83f-479e-b448-7bc98a74ac48.png)
